cmd/cmd.go

```diff
@@ -48,6 +48,9 @@ func New() *cobra.Command {
 		newBootnodeCmd(RunBootnode),
 		newCreateClusterCmd(runCreateCluster),
 		newDKGCmd(dkg.Run),
+		newCreateCmd(
+			newCreateDKGCmd(runCreateDKG),
```

Comment:
`charon create dkg` just doesn't feel right to me

Reply:
i mean you can `do` a dkg but `creating` a dkg doesn't make sense to me

Reply:
I pushed for:
 - `charon define dkg` -> `cluster_definition.json`
 - `charon seal` -> `cluster_lock.json`
 - `charon run`
 
 Or for local cluster flow:
  - `charon define local`
  - `charon seal`
  - `charon run`
 
 But this this was rejected for:
  - `charon create dkg` -> `cluster_definition.json`
  - `charon dkg` -> `cluster_lock.json`
  - `charon run`

Reply:
is building a `cluster_definition` even part of a dkg?

Reply:
it's an input to a dkg, if i'm not wrong

Reply:
or we can do something like:
`charon create dkg-cluster`
or
`charon create dkg-config`

Reply:
Perhaps this should have been discussed as part of the ticket grooming. I don't think we should stop the merge of this because we do not agree on the command name. Let's create an issue and discuss it there the definete command name